### PR TITLE
COMMERCE-12243 | Order Detail Page to handle UOM

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/COMMERCE-12243

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderItemTableFDSView.java [hunk 1]
```diff
@@ -49,6 +49,8 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"discount", "discount"
 		).add(
 			"formattedQuantity", "quantity"
+		).add(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderItemTableFDSView.java:52]: this I believe should be under feature flag

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PlacedCommerceOrderItemTableFDSView.java [hunk 2]
```diff
@@ -49,6 +49,8 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"discount", "discount"
 		).add(
 			"formattedQuantity", "quantity"
+		).add(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PlacedCommerceOrderItemTableFDSView.java:52]: this I believe should be under feature flag

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM
Thank you!

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:stable

**Comment by baharturk** [comment]
ci:test:stable

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
relevant failures are not related to the PR. It can be force forwarded

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by CrescenzoRegaLF** [comment]
sent here
https://github.com/brianchandotcom/liferay-portal/pull/141407

**Pull request closed without merging**